src/webDriver_Arch/OperaDriver.java

```diff
@@ -0,0 +1,55 @@
+package webDriver_Arch;
+
+public class OperaDriver implements WebDriver {
+	
+	public OperaDriver() {
+
+		System.out.println("Launching opera browser");
+	}
+
+	@Override
+	public void click(String element) {
+		System.out.println("clicking on the element" + element);
+
+	}
+
+	@Override
+	public String getUrl(String url) {
+		
+		System.out.println("Launching the url "+ url);
+		
+		return url;
+		
+	}
+
+	@Override
+	public String getTitle(String title) {
+		System.out.println("The title is "+ title);
+		
+		return title;
+	}
+
+	@Override
+	public void sendKeys(String value) {
+		System.out.println("Enter the value in field " + value);
+	}
+
+	@Override
+	public void findElement(String ele) {
+		System.out.println("Finding the element " + ele);
+
+	}
+
+	@Override
+	public void findElements(String ele) {
+
+		System.out.println("Finding the elements " + ele);
+
+	}
+
+	@Override
+	public void quit() {
+		System.out.println("Closing the browser.");		
+	}
+
```

Comment:
Please re format your code.

Reply:
Hi GaganReviewer,

I have reformatted the code as per your comments.

Thanks,
GaganReq